### PR TITLE
I32 deployment setup the compose files for production environment

### LINKED ISSUE #32: Deployment: Setup the compose files for production environment
**Is your feature request related to a problem? Please describe.**
Setup the compose files and environment variables to setup production environment

**Describe the solution you'd like**
The compose file will pull the image defined as latest by #31. The environment variable can be defined in this stage

**Additional context**
The services to run are:
- backend
- database
- nginx (specifically to serve the admin files for django). see https://github.com/codersforcauses/wais/blob/main/docker-compose.prod.yml "client" 

If we have frontend hosted in digital ocean,
- frontend docker container (to be decided)
- reverse proxy (preferably traefik)

Optional:
- watchtower if possible (for automatic refresh of deployment).

### PR DESCRIPTION
**Pull request by zhanerken**
## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related Issue

- Resolve #32

### CODE REVIEW

File: docker-compose-prod.yml [hunk 1]
```diff
@@ -13,30 +13,38 @@ services:
       retries: 5
```
Comment (frinzekt) [inline docker-compose-prod.yml:13]: we need 2 files:
- `docker-compose.yml` for local development
- `docker-compose-prod.yml` for production deployment.

**Review by jsun1590 — APPROVED** [review]
LGTM

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -13,30 +13,38 @@ services:
...
+      - server
+
+  watchtower:
+    image: containrrr/watchtower
+    container_name: csf_watchtower
+    restart: unless-stopped
     volumes:
-      - ./client:/app
+      - /var/run/docker.sock:/var/run/docker.sock
+    command: --interval 30
```
Comment (ChantelleKerr) [inline docker-compose-dev.yml:50]: This shouldnt use dockerhub, it build locally

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Small mistake

**Comment by frinzekt** [comment]
You should retain the docker-compose.yml file ... Don't rename it... If you want to rename it, then you'll have to tell everyone the syntax to run the new compose file, which you don't have to if you dont

File: docker-compose.yml [hunk 3]
```diff
@@ -22,9 +22,6 @@ services:
     build:
       context: .
       dockerfile: ./docker/server/Dockerfile
-    depends_on:
```
Comment (frinzekt) [inline docker-compose.yml:25]: what's the reason for this change?

File: docker/client/yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (frinzekt) [inline docker/client/yarn.lock:1]: this file should not exist

**Review by frinzekt — APPROVED** [review]
looks good to me to be merged, you can follow it up with a cleanup pr

**Pull request merged**